### ISSUE TITLE
prisma.model.createMany is not a function

### ISSUE THREAD

**Issue by Shigetorum635** [prisma/prisma#11507]
### Bug description

From what I was reading in the documentation, prisma supports `prisma.createMany({})` since version 2.2 or so.

```ts
model.createMany({
data: []
})`
```

for some reason it does not work on my 3.8?
` "prisma": "^3.8.1"`

```js
prisma.category.createMany({
    data: [
        {
            categoryName: "Anomia",
            categoryDescription: "General on-topic threads."
        },
        {
            categoryName: "Privacy",
            categoryDescription: "Any conversation privacy-wise related."
        },
        {
            categoryName: "Linux",
            categoryDescription: "Linux / Open Source Operative Systems related."
        },
        {
            categoryName: "Crypto / ETH",
            categoryDescription: "Cryptocurriencies / Etheremu related. (Includes NFT's)"
        },
        {
            categoryName: "Gaming",
            categoryDescription: "Game development / Gaming related."
        },
        {
            categoryName: "FOSS",
            categoryDescription: "Any FOSS related threads. Releases, requests etc."
        },
        {
            categoryName: "Off Topic",
            categoryDescription: "Anything can go here. Just abide the rules."
        }

    ]
}).then(() => {
    conmsole.log(`Done`)
})
```
![image](https://user-images.githubusercontent.com/48862659/151759071-589483c7-f3ec-492e-97a4-fcf47f25e61c.png)
![image](https://user-images.githubusercontent.com/48862659/151759095-2dee07e6-3d3d-49d7-8182-b1bb849834ea.png)

I get this error instead:

```
PS C:\Users\user\Development\Cool Projects\Site or something> node .\prisma\factory.js
C:\Users\user\Development\Cool Projects\Site or something\prisma\factory.js:5
prisma.category.createMany({
                ^

TypeError: prisma.category.createMany is not a function
    at Object.<anonymous> (C:\Users\user\Development\Cool Projects\Site or something\prisma\factory.js:5:17)
    at Module._compile (node:internal/modules/cjs/loader:1097:14)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47

```

### How to reproduce

1. Create a new Prisma 3.8.1 Project
2. Create a model that has 2 String columns
3. Create another model that's related to it (Array of the parent)

```js
Category -> Thread[]
```

try using createMany()

### Expected behavior

To detect the method and create the different rows.

### Prisma information

```ts
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "sqlite"
  url      = "file:database.sqlite"
}

model User {
  id              Int               @id @default(autoincrement())
  username        String            @unique
  password        String
  description     String            @default("I'm new!")
  status          String            @default("Doing poggers")
  isStaff         Boolean           @default(false)
  createdAt       DateTime          @default(now())
  responses       Responses[]
  threads         Threads[]
  ratings         ProviderRatings[]
  episodeComments EpisodeComments[]
}

model Threads {
  id         Int         @id @default(autoincrement())
  category   Category    @relation(fields: [categoryId], references: [id])
  categoryId Int
  title      String
  content    String
  author     User        @relation(fields: [authorId], references: [id])
  authorId   Int
  responses  Responses[]
}

model Responses {
  id       Int     @id @default(autoincrement())
  thread   Threads @relation(fields: [threadId], references: [id])
  threadId Int
  author   User    @relation(fields: [authorId], references: [id])
  authorId Int
}

model Category {
  id                  Int       @id @default(autoincrement())
  categoryName        String
  categoryDescription String
  threads             Threads[]
}

```

Removeds some of the other models as they are not needed.

### Environment & setup

- OS: Windows 10 21H1
- Database: SQLite
- NodeJS Version: 17.3.1

### Prisma Version

```
PS C:\Users\user\Development\Cool Projects\Site or something> npx prisma -v
Environment variables loaded from .env
prisma                  : 3.8.1
@prisma/client          : 3.8.1
Current platform        : windows
Query Engine (Node-API) : libquery-engine 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules\@prisma\engines\query_engine-windows.dll.node)
Migration Engine        : migration-engine-cli 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules\@prisma\engines\migration-engine-windows.exe)
Introspection Engine    : introspection-core 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary           : prisma-fmt 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Default Engines Hash    : 34df67547cf5598f5a6cd3eb45f14ee70c3fb86f
Studio                  : 0.452.0
```

**Comment by Shigetorum635** [comment]
As a workaround I just made an array.
```js
categories.forEach(async (category) => {
    console.log(`Creating category ${category.categoryName}...`);
    await prisma.category.create({
        data: {
            categoryName: category.categoryName,
            categoryDescription: category.categoryDescription
        }
    })
    console.log(`Category ${category.categoryName} created!~`);

})
```

**Comment by janpio** [comment]
`createMany` is not supported on SQLite unfortunately: https://github.com/prisma/prisma/issues/10710
Documented here: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#remarks-9

Should we add this somewhere else in the docs to make this more discoverable?

**Comment by Shigetorum635** [comment]
> `createMany` is not supported on SQLite unfortunately: #10710 Documented here: https://www.prisma.io/docs/reference/api-reference/prisma-client-reference#remarks-9
> 
> Should we add this somewhere else in the docs to make this more discoverable?

Thanks for the answer, and yes it'd come handy in the docs, I was wondering why I didnt have it haha.

Any info on why its not supported? There could be a workaround like the one I did above. Many thanks & Closing.

**Comment by janpio** [comment]
Probably the necessity of the workaround and it is not supported out of the box.

**Comment by wilkinnovo** [comment]
I'd be imperative to do an implementation for Sqlite as we have turso.tech now.

**Comment by janpio** [comment]
Us supporting Turso indeed makes #10710 more interesting, so we will be taking another look at what we can do there when we have the capacity.

**Comment by wilkinnovo** [comment]
> Us supporting Turso indeed makes #10710 more interesting, so we will be taking another look at what we can do there when we have the capacity.

No rush. :+1:

**Comment by hyper-dot** [comment]
Great :+1:  #10710

**Comment by rs-nikic** [comment]
I'm using postgreSQL, still facing the same issue.

```
datasource db {
  provider = "postgresql"
  url      = env("PG_CONNECTION_STRING")
}

model node_locations {
  NodeId String @unique(map: "node_locations_id_idx") @default(dbgenerated("gen_random_uuid()")) @db.Uuid
  ExternalId String @unique(map: "node_locations_ext_id_idx")
  Name String
  Description String
  Geometry Unsupported("geometry(Point, 4326)")
  Meta Json
  Provider String
  Species String[]
  Status String
  LastPing DateTime

  @@index([Geometry], name: "node_locations_geometry_idx", type: Gist)
}
```

Prisma version: 5.5.2

Note: `prisma db push` is used for code generation 

![image](https://github.com/prisma/prisma/assets/143053594/81a1b7d1-02e1-4591-82f7-2829b147883c)

**Comment by jgb-solutions** [comment]
Yes it would be nice to give SQLite a little more love these days. I found myself waiting a long time today trying to seed a turso database with prisma. I had to `create` thousands of records one by one in a loop. Drizzle worked like a charm, allowing to add either an object or an array to the `values` method. I really the syntax and DX of Prisma, but Drizzle is catching up, fast.  +1 for this issue.
